src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let discount = 0;
+  const priceForDay = 40;
+  const count = days * priceForDay;
+
+  if (days >= 3) {
+    discount = 20;
```

Comment:
The discount values should be defined as constants at the top of the file to avoid hardcoding and to make the code more maintainable.